installer/bundle/bundle_skel_Linux.sh

```diff
@@ -702,29 +581,21 @@ case "$installMode" in
 
     I)
         echo "Installing cross-platform agent ..."
-        if [ "$PLATFORM" = "SunOS" ]; then
-            check_if_pkg_is_installed scx
-            if [ $? -eq 0 ]; then
-                echo "ERROR: SCX package is already installed"
-                cleanup_and_exit 2
-            fi
-        fi
 
         if [ $PROVIDER_ONLY -eq 0 ]; then
             check_if_pkg_is_installed omi
             if [ $? -eq 0 ]; then
-                pkg_upd $OMI_PKG omi
-                # It is acceptable that this fails due to the new omi being 
-                # the same version (or less) than the one currently installed.
-                OMI_EXIT_STATUS=0
+                pkg_upd_list $OMI_PKG omi
+                pkg_upd ${UPD_PKG_QUEUE}
             else
-                pkg_add $OMI_PKG omi
-                OMI_EXIT_STATUS=$?
+                pkg_add_list $OMI_PKG omi
             fi
         fi
 
-        pkg_add $OM_PKG scx
-        SCX_EXIT_STATUS=$?
+        pkg_add_list $OM_PKG scx
+
+        pkg_add ${ADD_PKG_QUEUE}
+        SCX_OMI_EXIT_STATUS=$?
 
         if [ $PROVIDER_ONLY -eq 0 ]; then
```

Comment:
Thanks for patiently explaining this. I understand it better now. I'll look into the impact on configure and make.  I will try to club the changes for these with the PR for SunOS skeleton.